juniper_official/Solutions/EVPN-VXLAN/evpn-vxlan-monitor-count-online-netconf.rule

```diff
@@ -0,0 +1,204 @@
+/*
+ * Collects NHCount of fpc and notifies anomalies when NHCount is more then
+ * provided threshold. Also check-vport-count monitor vport count and notifies
+ * anomalies when vport count is more then provided threshold value.
+ *
+ * One input control detection
+ *
+ *  1) high-threshold-count, is the NHCount maximum threshold value.
+ *
```

Comment:
s/high-threshold-count,/high-threshold-count variable,/g

Reply:
updated the rule